ankerctl.py

```diff
@@ -32,10 +32,13 @@ class Environment:
     def __init__(self):
         pass
 
-    def require_config(self):
+    def require_config(self, required=True):
         with self.config.open() as config:
             if not getattr(config, 'printers', False):
-                log.critical("No printers found in config. Please import configuration using 'config import'")
+                if not required:
```

Comment:
That part is fine, and reasonable. I'm just saying we should use the same error message in both cases, even if the level is different.

Reply:
But this function is used in multiple places, if you aren't planning on using the webserver, why would we tell you to import the config via the web interface and vice-versa, if you are using the webserver and not the CLI functions, why tell you to use `config import`?

Reply:
I just think it's helpful to tell the user that they have both options, since the cli and webserver currently share the same config file.

But, more than that, it seems to me that we are really forcing two different `load_config` functions into the same function body. We could also write two functions, and have one fail on missing config (`log.critical`), and the other use `log.info()` on missing config. Then it would make sense for them to have different error messages.

But this style, where one function does two different jobs depending on a parameter, is kind of confusing.